src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/UpDownBaseTests.cs

```diff
@@ -3032,6 +3032,13 @@ public void UpDownBase_Invokes_SetToolTip_IfExternalToolTipIsSet()
 
         Assert.Equal(text, actualEditToolTipText);
         Assert.Equal(text, actualButtonsToolTipText);
+
+        toolTip.SetToolTip(upDownBase, null); // Invokes UpDownBase's SetToolTip inside
+        actualEditToolTipText = toolTip.GetToolTip(upDownBase._upDownEdit);
+        actualButtonsToolTipText = toolTip.GetToolTip(upDownBase._upDownButtons);
+
+        Assert.Equal("", actualEditToolTipText);
```

Comment:
```suggestion
        Assert.Equal(string.Empty, actualEditToolTipText);
```

Ditto below as well. Could we go ahead and change this test to use fluent assertions since we are already here? (no need to do the entire file, unless you would like to of course :) )

Reply:
I've pushed the change to fluent assertions for the test.